tests/db/test_relations.py

```diff
@@ -88,89 +74,7 @@ def test_relations(self):
                 self.assertIn(order.pk, contactrows[contact.pk])
 
     def test_bad_many_to_many(self):
-        db = self.db
-
         with self.assertRaises(sa.exc.ArgumentError):
 
             class SuperDummy(db.Model):
                 bad = db.ManyToMany("blah")
-
-
-class TestSQLAlchemyRelationshipsBackPopulates(TestCase):
-    def setUp(self):
-        super(TestSQLAlchemyRelationshipsBackPopulates, self).setUp()
-
-        from ..models_backpop import db, Asset, Customer, Order, Contact
-
-        self.db = db
-        self.asset = Asset
-        self.customer = Customer
-        self.order = Order
-        self.contact = Contact
-
-        db.Model.metadata.create_all(bind=db.engine)
-
-    def test_relations(self):
```

Comment:
not sure why but this test case is direct duplicate of test case above. literally word to word

Reply:
cause its a separate app, db and schema declared with `back_populates` instead of `backref`